### PR TITLE
Adding padelpy

### PR DESCRIPTION
**Pull request by maclandrol**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/padelpy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ecrl/padelpy
+  summary: A Python wrapper for PaDEL-Descriptor
+  license: AGPL-3.0-only
+  license_file: padelpy/PaDEL-Descriptor/license/LICENSE.txt
```
Comment (dopplershift) [inline recipes/padelpy/meta.yaml:38]: It appears that license is one of several that apply to the javascript, but then the Python code is MIT. So what you need to do is add the [Python code's license]() to this PR in the recipe directory. And assuming you call it `LICENSE`, list all the licenses under `license_file`:
```suggestion
  license: AGPL-3.0-only AND LGPL-2.1-only AND MIT
  license_file:
    - padelpy/PaDEL-Descriptor/license/LICENSE.txt
    - padelpy/PaDEL-Descriptor/license/lgpl-2.1.txt
    - padelpy/PaDEL-Descriptor/license/lgpl.license
    - LICENSE
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Need a bit more on the license front. Also, we need confirmation from @tjkessler that they are willing to be listed as a maintainer for this recipe.

File: recipes/padelpy/meta.yaml (hunk 1 above)
Comment (maclandrol) [reply recipes/padelpy/meta.yaml:38]: Thanks @dopplershift. 

The LICENCE file is currently not packaged in the source code (waiting for the PR in the repo to fix it).

Comment (dopplershift) [reply recipes/padelpy/meta.yaml:38]: @maclandrol Right, you can work around that fact by committing the file to the `recipes/padelpy` directory in this PR.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**